spirv_cross_c.h

```diff
@@ -257,15 +257,24 @@ typedef enum spvc_msl_platform
 	SPVC_MSL_PLATFORM_MAX_INT = 0x7fffffff
 } spvc_msl_platform;
 
+/* Maps to C++ API. */
+typedef enum spvc_msl_index_type
+{
+	SPVC_MSL_NO_INDEX = 0,
+	SPVC_MSL_INDEX_16 = 1,
+	SPVC_MSL_INDEX_32 = 2
```

Comment:
Must have a SPVC_MSL_INDEX_TYPE_MAX_INT to force 32-bit enum.

Reply:
Ugh, it's not like we have to build 16-bit code--that's why DX had those `FORCE_DWORD` enums in the first place. All right, fine... /me grumbles